portfolio/src/main/webapp/functions/index.js

```diff
@@ -1,11 +1,26 @@
 const createCommentElement = commentPayload => {
     const username = document.createElement("span");
+    const sentiment = document.createElement("span");
     const comment = document.createElement("p");
     const deleteOption = document.createElement("i");
 
     username.innerHTML = `@ ${commentPayload.username}`;
     username.classList.add("badge", "badge-dark");
     comment.innerHTML = commentPayload.comment;
+
+    const score = parseFloat(commentPayload.sentiment);
+    const emotion =
+        score >= -1 && score <= -0.3 ? "Mad" :
+        score > -0.3 && score <= 0.3 ? "Neutral" :
+        "Happy";
```

Comment:
Consider moving the logic to translate the score to a string to the backend. That way, the frontend only needs to worry about display logic, not business logic.